src/Services/UnitTestLogger.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Symbiote\QueuedJobs\Services;
+
+use Psr\Log\LoggerInterface;
+use SilverStripe\Dev\TestOnly;
+
+/**
+ * Class UnitTestLogger
+ *
+ * This logger is meant to be used only for Unit tests which produce unwanted console output
+ * this often breaks test suites like code coverage even though unit tests themselves pass
+ *
+ * @package Symbiote\QueuedJobs\Services
+ */
+class UnitTestLogger implements LoggerInterface, TestOnly
```

Comment:
Can just use `TestCase::createMock(LoggerInterface::class)` instead. Let's remove this class.